src/convertToObject.js

```diff
@@ -5,8 +5,25 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
-}
+const convertToObject = (stylesString) => {
+  const styles = {};
+
+  const declarations = stylesString.split(';');
+
+  declarations.forEach((declaration) => {
+    const trimmedDeclaration = declaration.trim();
+
+    if (trimmedDeclaration) {
+      const [property, value] = trimmedDeclaration.split(':');
+
+      const trimmedProperty = property.trim();
+      const trimmedValue = value.trim();
```

Comment:
There's a potential bug here. If `value` is undefined due to a missing colon in the declaration, calling `trim` on it will throw an error. It's important to check if `value` exists before attempting to trim it.